WordPress/src/main/res/layout/activity_log_list_event_item.xml

```diff
@@ -1,39 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
               xmlns:tools="http://schemas.android.com/tools"
+              android:id="@+id/activity_content_container"
               android:layout_width="match_parent"
               android:layout_height="wrap_content"
-              xmlns:app="http://schemas.android.com/apk/res-auto"
+              android:background="?android:selectableItemBackground"
```

Comment:
There is no touch feedback when tapping a list item since the child `LinearLayout` has `android:background="@color/white"`.  We can change `android:background` to `android:foreground` in the parent `LinearLayout` for a quick fix, but I prefer we rethink this layout for a long-term solution.

Reply:
I've switched the order, the container has a white background now and the LL inside has the selectableItemBackground. It seems to work 👍